src/app/user/feedback/page.js

```diff
@@ -0,0 +1,17 @@
+import ProtectedPage from "@/components/dynamic/ProtectedPage";
+import Feedback from "@/components/dynamic/forms/Feedback";
+
+const Page = () => {
```

Comment:
this page is already created and available at /forms/feedback, this link should simply redirect to that links